scheduler/src/cook/mesos.clj

```diff
@@ -25,8 +25,10 @@
             [cook.datomic :refer (transact-with-retries)]
             [cook.mesos.heartbeat]
             [cook.mesos.monitor]
+            [cook.mesos.optimizer]
             [cook.mesos.rebalancer]
             [cook.mesos.scheduler :as sched]
+            [cook.mesos.util]
```

Comment:
Can we alias this to `mutil` and use that to qualify accesses to the functions?

Reply:
I kept the same pattern as the rest of the imports in this file. If we want to change it, can it be in a different PR so this one doesn't get too big?